### DISCUSSION TITLE
Configuring Static IP on AI-Deck

### DISCUSSION THREAD

**Discussion by yanrui89** [bitcraze/discussions#533] [Q&A] [answered]
May I know how to configure Static IP on the WiFi module on the AI-Deck? Thanks

**Comment by Kaijun101** [comment, marked as answer]
@yanrui89 have u managed to solve this? or @hmllr or @evoggy any insights to this hahaha?

**Reply by hmllr** [reply]
Sorry, we don't have much insight on this - this is something you'll need to change on the NINA, so it is more an ESP32 question than a Crazyflie question. I did this for another project on an ESP32 before, with the menuconfig for espressif chips in a vscode plugin (like here: https://esp32tutorials.com/esp32-static-fixed-ip-address-esp-idf/ which then autogenerates the needed code), but I don't think you can directly apply this to the existing NINA code. However I haven't tried and maybe even if it does not work out of the box you could use it as example code to integrate this into the existing code.
Hope this at least helps a bit,
Hanna

**Reply by Kaijun101** [reply]
@hmllr thanks for the tips, will try it out!
